docs/index.rst

```diff
@@ -1,6 +1,10 @@
 TestSlide: Fluent Python Testing
 ================================
 
```

Comment:
I think we can drop this now that the logo is there, right?

Reply:
I think having the "Fluent Python Testing" is nice, but we could drop it